fastly/wasm.go

```diff
@@ -0,0 +1,96 @@
+package fastly
+
+import (
+	"fmt"
+	"io"
+	"time"
+)
+
+type WASMPackageMetadata struct {
```

Comment:
Just noting what @phamann pointed out in another place

https://webassembly.github.io/spec/core/intro/introduction.html?highlight=wasm#id1
```suggestion
type WasmPackageMetadata struct {
```

Reply:
This should be resolved.